projects/frontend/data-pipelines/gui/e2e/README.md

```diff
@@ -0,0 +1,50 @@
+## Setup for running Cypress tests on local environment ##
```

Comment:
How does that differ from https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/data-pipelines/gui/README.md#running-and-testing ?

Reply:
I didn't know there is such README file, I'm missing pass with the changes in this library, and I think we should discuss about code ownership and who is responsible for the UI project, because a lot of things are happening.

Reply:
The README is linked in VEP-1507 https://github.com/vmware/versatile-data-kit/tree/main/specs/vep-1507-vdk-operations-ui#glossary
The source of truth is the VDK enhancement proposal specification, see the contributing guide https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#propose-a-change

Reply:
The ask here is, to read the linked README and adjust this README change accordingly? May consider adding to the already existing readme only the information missing information, or adding the missing info in this separate file and why